oneview_redfish_toolkit/blueprints/storage.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Python libs
+import logging
+
+# 3rd party libs
+from flask import abort
+from flask import Blueprint
+from flask import Response
+from flask_api import status
+
+# own libs
+from hpOneView.exceptions import HPOneViewException
+from oneview_redfish_toolkit.api.storage import Storage
+from oneview_redfish_toolkit import util
+
+storage = Blueprint("storage", __name__)
+
+
+@storage.route("/redfish/v1/Systems/<uuid>/Storage/1", methods=["GET"])
+def get_storage(uuid):
+    """Get the Redfish Storage for a given UUID.
+
+        Return Storage Redfish JSON for a given hardware UUID.
+        Logs exception of any error and return abort(500)
+        Internal Server Error.
+
+        Returns:
+            JSON: Redfish json with Storage
+            When hardware or hardware type is not found calls abort(404)
+
+        Exceptions:
+            Logs the exception and call abort(500)
+
+    """
+    try:
+        ov_client = util.get_oneview_client()
+
+        server_hardware = ov_client.server_hardware. \
+            get(uuid)
+        if server_hardware:
+            sht_uri = server_hardware['serverHardwareTypeUri']
```

Comment:
use full name?

Reply:
I think this name is probably fine the way it is.